source/_integrations/nest.markdown

```diff
@@ -162,15 +182,21 @@ subscriber_id:
 
 Once your developer account is set up and you have a valid `nest` entry in `configuration.yaml`, you need to connect devices with the following steps:
 
-1. From the Home Assistant front-end, navigate to **Configuration** then *Integrations**. Under **Set up a new integration** locate 'Nest'.
+1. From the Home Assistant front-end, navigate to **Configuration** then **Integrations**.  Click **Add Integration** then locate 'Nest'.
 
 1. You should get redirected to Google to choose an account. This should be the same developer account you configured above.
 
-1. The *Google Nest permissions* screen will allow you to choose which devices to configure. You can leave out any device that you do not wish to use with Home Assistant.
+1. The *Google Nest permissions* screen will allow you to choose which devices to configure.  You likely want to enable
+everything, however you can leave out any feature you do not wish to use with Home Assistant.
+
     ![Screenshot of Nest permissions authorization](/images/integrations/nest/oauth_approve.png)
 
 1. You will get redirected back to another account selection page. See [Troubleshooting](#troubleshooting) below if you get a `redirect_uri_mismatch` error.
 
+1. You may see a warning screen that says *Google hasn't verified this app* since you just set up an un-verified developer workflow.  Click *Advanced* then *Go to yourdomain (unsafe)* to proceed.
```

Comment:
```suggestion
1. You may see a warning screen that says *Google hasn't verified this app* since you just set up an un-verified developer workflow. Click *Advanced* then *Go to your domain (unsafe)* to proceed.
```